.buildbot.sh

```diff
@@ -94,6 +94,33 @@ for i in $(seq 10); do
     YKD_NEW_CODEGEN=1 RUST_TEST_SHUFFLE=1 cargo test
 done
 
+# Test with LLVM sanitisers
+RUSTFLAGS="-Z sanitizer=address" cargo test --target x86_64-unknown-linux-gnu
+# The thread sanitiser has a number of false positives by default, so we need a
+# suppression file to avoid those overwhelming us. This does mean that we might
+# suppress some true positives, but there's little we can do about that.
+suppressions_path=`mktemp`
+cat << EOF > $suppressions_path
+# The tests::EXTRA_LINK closure leads to a race being reported, but that seems
+# to be due to an internal detail of `LazyLock` rather than our code.
+race:tests::EXTRA_LINK::
```

Comment:
The EXTRA_LINK stuff invokes the compiler without any of `yk-config`s flags. If we tack on the stuff required for the sanitiser, does it silence some of the spurious errors?

Reply:
I don't understand the question.

Reply:
Do you need to add sanitiser flags here: https://github.com/ykjit/yk/blob/ec757fefde04c6b8df62605168922a71b0da884c/tests/src/lib.rs#L37 ?

Reply:
I don't understand this question either!

Reply:
Do you need to add snaitiser flags to invocations of the C compiler for the code that will be linked to the rust code?